charts/operator-wandb/charts/weave/templates/deployment.yaml

```diff
@@ -63,6 +63,12 @@ spec:
               value: {{ .Values.global.host }}
             - name: WEAVE_SERVER_NUM_WORKERS
               value: "4"
+            - name: DD_SERVICE
+              value: weave-python
```

Comment:
I think you want this wrapped in a 
```helm
{{- if .Values.datadog.enabled }}
{{- end }}
```
Otherwise you will always be setting this.

Reply:
done. had to add the value too